docs/control_center_docs/speed_module_getting_started.rst

```diff
@@ -0,0 +1,166 @@
+Getting Started with Vertiq's Speed Firmware with IQ Control Center
+####################################################################
+
+Before completing the following getting started guide, please ensure that you have read and completed our :ref:`IQ Control Center 
+guide <control_center_start_guide>`. It walks you through Control Center installation, module configuration, and basic testing options available 
+to all Vertiq modules and firmware types. The following document is meant to provide additional configurations and testing options available only on Vertiq's speed firmware.
+
+.. note::
+    The following images are captured using IQ Control Center version 1.5.2 as connected to a Vertiq 40-06 Gen 2
+
+*********************************************************
+Important Parameters for Flight Controller Integration
+*********************************************************
+
+The first set of configurations covered here are those that **must** be configured properly in order to control your module with a flight controller. 
+These parameters affect how your module communicates with, and is controlled by a flight controller. 
+
+* **Mode**: this parameter is available through the General tab, and defines whether your module accepts throttle commands as a voltage, a velocity, or a PWM. 
+  To learn more about the Mode parameter see :ref:`throttle_mode`
+* **Direction**: this parameter is available through the General tab, and defines the direction that your module takes to be positive, either clockwise or 
+  counter-clockwise. You can also configure either 2D or 3D configuration to allow for mapping negative throttle commands.
+  For more on the *Direction* parameter and throttle mapping, see :ref:`throttle_direction`
+* **FC 2D/3D Mode**: This parameter is available through the General tab, and defines how ESC commands will be mapped to a throttle. To learn more see :ref:`throttle_mapping`
+* **Communication**: This parameter is available through the General tab, and defines the communication protocol expected by your module as sent by the 
+  flight controller. This parameter can be Standard PWM, OneShot protocols, MultiShot, or DShot protocols. Please note that if you are 
+  controlling your module with :ref:`DroneCAN <dronecan_protocol>` or :ref:`IQUART <uart_messaging>`, this parameter **does not** apply. To learn more see :ref:`throttle_sources`
+* **Max Velocity and Max Volts**: These parameters are available through the Tuning tab, and pair directly with the Mode parameter. This means that when your 
+  mode is set to velocity, your module will only ever map a throttle command as bounded by ± *Max Velocity*. When your mode is set to voltage, your module 
+  will only ever map a throttle as bounded by ± *Max Volts*. To learn more see :ref:`throttle_maximums`
+
+The following set of configurations covered here are those that **can optionally** be configured in order to further customize your flight controller integration.
```

Comment:
Looking at it now, why are we singling out arming for discussion here? You have a separate section on Arming and Disarming Configuration below, so why not move this stuff down there? It does seem a bit odd to discuss only the arming and disarming stuff here, as there are plenty of other things that could be important for flight controller integration, like DroneCAN or stow stuff, which you cover in the additional sections down below. 

Did you call these ones out separately here especially because you were going to use them in the example? If so, maybe it would make sense to move this to the arming/disarming configuration section, and then to just bring up those extra configuration sections before the example? You could have all the sections on relevant configurations, then the example, and then finish with Next Steps